modules/nw-ingress-gateway-api-enable.adoc

```diff
@@ -0,0 +1,65 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-gateway-api-enable_{context}"]
+= Enabling Gateway API for the Ingress Operator
```

Comment:
Is this under a section for Gateway API?  I think a better title might be "Getting Started with Gateway API".  But I guess it depends on the surrounding content.